app/models/polymorphic/task.rb

```diff
@@ -34,7 +34,7 @@ class Task < ActiveRecord::Base
   belongs_to :completor, class_name: "User", foreign_key: :completed_by
   belongs_to :asset, polymorphic: true
 
-  serialize :subscribed_users, Set
+  serialize :subscribed_users, Array
```

Comment:
As far as I can tell it was always serialized as an array and this only effects how it behaves but we should ensure we are not appending to subscribed_users without checking for existing.